### PR TITLE
Small documentation fixes.

### PR DESCRIPTION
**Pull request by MagneFire**
This PR contains some of the overdue documentation fixes that I forgot to actually fix.
The commits should explain it quite well :wink:

### CODE REVIEW

File: pages/wiki/creating-an-asteroid-app.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ layout: documentation
 
 ---
 
-The app creation process of AsteroidOS needs a Software Development Kit generated by OpenEmbedded. You can either grab a prebuilt SDK for the nightlies [here](https://release.asteroidos.org/nightlies/sdk/oecore-x86_64-armv7vet2hf-neon-toolchain-nodistro.0.sh) and install it on your system or you can build it yourself as follows:
+The app creation process of AsteroidOS needs a Software Development Kit generated by OpenEmbedded. You can either grab a prebuilt SDK for the nightlies [here](https://release.asteroidos.org/nightlies/sdk/oecore-x86_64-armv7vehf-neon-toolchain-nodistro.0.sh) and install it on your system or you can build it yourself as follows:
```
Comment (jrtberlin) [inline pages/wiki/creating-an-asteroid-app.md:11]: We use the wrong arch throughout this file. For example in L48, L54,...
Can you change it to the corect one in the whole file please?

Comment (MagneFire) [reply pages/wiki/creating-an-asteroid-app.md:11]: Didn't even think about it. Thanks!

Comment (jrtberlin) [reply pages/wiki/creating-an-asteroid-app.md:11]: No worries, that's what reviews are for :D

**Comment by jrtberlin** [comment]
Thanks

**Pull request merged**